.semgrep/semgrep.yml

```
@@ -0,0 +1,172 @@
+rules:
+  # 1) Repository 계층: 허용된 메서드 접두어만 사용
+  - id: repo-method-naming-allowed-prefixes
+    languages: [java]
+    message: "Repository 메서드는 findBy*/countBy*/existsBy*/insert*/update*/delete* 접두어만 허용됩니다."
+    severity: ERROR
+    patterns:
+      - pattern: |
+          class $C {
+            ... 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)